src/mongoc/mongoc-client.c

```diff
@@ -2185,6 +2191,7 @@ _mongoc_client_monitor_op_killcursors_failed (
    client->apm_callbacks.failed (&event);
 
    mongoc_apm_command_failed_cleanup (&event);
+   bson_destroy(&doc);
```

Comment:
Space before paren - you can read the "C Driver Development" guide in Google docs if you want to see how to run clang-format, or just manually format the code according to our style.